README.md

```diff
@@ -122,46 +122,20 @@ This is useful if you want to authenticate as an IAM user directly or if you wan
-   ```
-2. Add the following sections to the cluster spec, either using `kops edit cluster ${CLUSTER_NAME}` or editing the manifest yaml file.
-   Be sure to replace `KOPS_STATE_STORE` and `CLUSTER_NAME` with their appropriate values since those environment variables are not available at runtime.
-   This downloads the files from the state store on masters to a directory that is volume mounted by kube-apiserver.
-   Kops does not support adding additional volumes to kube-apiserver so we must reuse the existing `/srv/kubernetes` hostPath volume.
+Perform the following steps to setup Authenticator on a Kops cluster.
+Note that Kops 1.10 and Kubernetes 1.10 are required.
+For more information see the [Kops documentation](https://github.com/kubernetes/kops/blob/master/docs/authentication.md#aws-iam-authenticator).
+1. Add the following section to the cluster spec, either using `kops edit cluster ${CLUSTER_NAME}` or editing the manifest yaml file.
    ```
    apiVersion: kops/v1alpha2
    kind: Cluster
    spec:
-     kubeAPIServer:
-       authenticationTokenWebhookConfigFile: /srv/kubernetes/aws-iam-authenticator/kubeconfig.yaml
-     hooks:
-     - name: kops-hook-authenticator-config.service
-       before:
-         - kubelet.service
-       roles: [Master]
-       manifest: |
-         [Unit]
-         Description=Download AWS Authenticator configs from S3
-         [Service]
-         Type=oneshot
-         ExecStart=/bin/mkdir -p /srv/kubernetes/aws-iam-authenticator
-         ExecStart=/usr/local/bin/aws s3 cp --recursive s3://KOPS_STATE_STORE/CLUSTER_NAME/addons/authenticator /srv/kubernetes/aws-iam-authenticator/
+     authentication:
+       aws: {}
    ```
-  If using a non-default AMI that does not have the AWS CLI, replace the second ExecStart statement with
 
-  ```
-  ExecStart=/usr/bin/docker run --net=host --rm -v /srv/kubernetes/aws-iam-authenticator:/srv/kubernetes/aws-iam-authenticator quay.io/coreos/awscli@sha256:7b893bfb22ac582587798b011024f40871cd7424b9026595fd99c2b69492791d aws s3 cp --recursive s3://KOPS_STATE_STORE/CLUSTER_NAME/addons/authenticator /srv/kubernetes/aws-iam-authenticator/
-  ```
-3. Apply the changes with `kops update cluster ${CLUSTER_NAME}`.
+2. Apply the changes with `kops update cluster ${CLUSTER_NAME}`.
    If the cluster already exists, roll the cluster with `kops rolling-update cluster ${CLUSTER_NAME}` in order to recreate the master nodes.
-4. Update the Authenticator DaemonSet's state and output volumes to both use `/srv/kubernetes/aws-iam-authenticator/` for their `hostPath`s.
-5. Apply the DaemonSet and ConfigMap resource manifests to launch the Authenticator server on the cluster.
 
 *Note:* Certain Kops commands will overwrite the `ExecCredential` in kubeconfig so it may need to be restored manually. See [kubernetes/kops#5051](https://github.com/kubernetes/kops/issues/5051) for more information.
```

Comment:
We should probably change this to say something like "If you are using Kops <1.10 …" instead of "Certain Kops" since the 1.10 Kops fixes this bug by supporting `client-go` 7.0

Reply:
It's kind of strange to provide instructions that require >=1.10 and then give a warning about behavior when using <1.10. Maybe we just remove the warning altogether if we're not going to provide instructions for <1.10 ?

Reply:
That's a fair point. Removing is probably the better option.